components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1405,7 +1410,8 @@ void RewardsServiceImpl::EnableGreaseLion() {
 
 void RewardsServiceImpl::OnRulesReady(
     greaselion::GreaselionService* greaselion_service) {
-  EnableGreaseLion();
+  EnableGreaselion();
+  greaselion_enabled_ = true;
```

Comment:
Shouldn't this line be the last line of `EnableGreaselion()`?

Reply:
Ah, great point - fixed!